src/Container/EventStoreFactory.php

```diff
@@ -1,12 +1,12 @@
 <?php
 /*
- * This file is part of the prooph/proophessor.
- * (c) 2014 - 2015 prooph software GmbH <contact@prooph.de>
+ * This file is part of the prooph/event-store.
+ * (c) Alexander Miertsch <contact@prooph.de>
```

Comment:
oh, good catch. The header of the event-store is an old one. The copyright of proophessor is the newer version. But I create an ticket to cover the issue.